### PR TITLE
Addresses missing `hidden` toctree attribute

### PR DESCRIPTION
**Pull request by Jegp**
Addresses #73 where files generated by apidoc crashes due to a missing attribute.

### CODE REVIEW

**Comment by xmnlab** [comment]
oh sorry .. I create a new PR for the same issue https://github.com/executablebooks/sphinx-jupyterbook-latex/pull/78
I just saw your PR ... I will close mine in favor of your.

**Comment by xmnlab** [comment]
the merge of this bugfix would be very appreciated :)

File: sphinx_jupyterbook_latex/transforms.py [hunk 1]
```diff
@@ -297,9 +297,8 @@ class LatexToctreeNodeInterpret(SphinxTransform):
 
     def apply(self, **kwargs: Any) -> None:
         for tocnode in self.document.traverse(toctree_node):
-            if tocnode.attributes["hidden"]:
+            if "hidden" in tocnode.attributes and tocnode.attributes["hidden"]:
```
Comment (xmnlab) [inline sphinx_jupyterbook_latex/transforms.py:300]: another way to write this would be:
```suggestion
            if tocnode.attributes.get("hidden"):
```

**Comment by mmcky** [comment]
thanks @Jegp and @xmnlab sorry it has taken so long to review this. Absolutely agree with this change -- thank you.

**Review by mmcky — APPROVED** [review]

**Comment by xmnlab** [comment]
That sounds great. Thanks @Jegp and @mmcky !

**Pull request merged**